lib/node/runner/isaac_state_manager.go

```diff
@@ -45,13 +45,13 @@ func NewISAACStateManager(nr *NodeRunner, conf common.Config) *ISAACStateManager
 		Conf:            conf,
 	}
 
-	p.firstConsensusBlockTime = time.Time{}
+	p.firstProposedBlockTime = time.Time{}
```

Comment:
@Charleslee522 Instead of setting current time, how about calling `setThefirstProposedBlockTime()`? 

If the first proposed block does not exist, `setThefirstProposedBlockTime()` set the current time.